presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetReader.java

```diff
@@ -58,7 +59,10 @@
         implements Closeable
 {
     private static final int MAX_VECTOR_LENGTH = 1024;
-    private static final String MAP_TYPE_NAME = "map";
+    //Per Parquet documentation - "Some existing data incorrectly used MAP in place of MAP_KEY_VALUE. For backward-compatibility,
```

Comment:
Can we link to the documentation here?

Reply:
nit: In general we put a space before a comment text: `//Per` -> `// Per`

Same for other comments below.